alpha/apps/kaltura/lib/myEntryUtils.class.php

```diff
@@ -1714,7 +1714,7 @@ public static function copyEntry(entry $entry, Partner $toPartner = null,
  		
  		KalturaLog::log("copyEntry - New entry [".$newEntry->getId()."] was created");
 
-		if ( $entry->getStatus() != entryStatus::READY ) {
+		if ( $entry->getStatus() != entryStatus::READY && $entry->getStatus() != entryStatus::NO_CONTENT ) {
```

Comment:
this fix can potentially break the behavior that it was built for - alpha/apps/kaltura/lib/batch2/kFlowHelper.php:2373